packages/react-inbox/src/components/Message/styled.tsx

```diff
@@ -107,7 +107,7 @@ export const UnreadIndicator = styled.div(({ theme }) =>
   deepExtend(
     {
       height: "36px",
-      backgroundColor: theme?.brand?.colors?.secondary ?? "#9121c2",
+      backgroundColor: theme?.brand?.colors?.tertiary ?? "#9121c2",
```

Comment:
slightly tangent, do we have a pattern to take this hex values and put in some common constant file maybe? if we ever decide to change this hex, we will have to look for all occurrences.

Reply:
yeah we should do that